app/views/acm/conference/2018.haml

```diff
@@ -0,0 +1,167 @@
+#{meta_tag "summary", :name => "twitter:card"}
```

Comment:
File comment: have you tested that it renders in a local instance?

Reply:
Are you  asking if the twitter link works. Or is there  something else we're suppose to load?

Reply:
Have you done: https://github.com/hacsoc/acm.case.edu/#instructions ? (local test)